Application/FileConverter/Settings.cs

```diff
@@ -207,6 +208,20 @@ public bool CheckUpgradeAtStartup
             }
         }
 
+        [XmlElement]
+        public bool CheckCopyFilesAfterConverting
```

Comment:
I'd prefer the naming 'CopyFilesInClipboardAfterConversion' to be more explicit.